<launch>


 <!-- <node pkg="ros_lane_control" type = "vision_xte_offset.py" name = "vision_xte_offset.py" output="screen"/> -->
 <node pkg="aa10_image_processing" type = "aa10_lane_detect_node_node" name = "aa10_lane_detect_node" output="screen">
 
<param name= "input_image_topic"                  type = "string"          value = "/camera/image1"/>     <!-- image topic -->
<param name= "img_width"                          type = "int"             value = "640"/>                <!-- image size height -->
<param name= "img_height"                         type = "int"             value = "360"/>    	      <!-- image size widht -->        
<param name= "lane_width"                         type = "double"          value = "560"/>    	      <!-- 100 -->
<param name= "ROI_CENTER_Y"                       type = "int"             value =  "120"/>   	      <!-- ROI center locaton -->          
<param name= "ROI_WIDTH"                          type = "int"             value =  "100"/>  	      <!-- ROI height size -->  
<param name= "perspective_upper"                  type = "int"             value =  "200"/>  	      <!-- bird eye roi upper value200 225-->
<param name= "perspective_lower"                  type = "int"             value =  "500"/>  	      <!-- bird eye roi lower value319 530-->      
<param name= "canney_low"                         type = "int"             value =  "110"/>  	      <!-- canney edeg parameter  -->
<param name = "Speed"					type = "double"      value = "1.0"/>
<param name= "canney_high"                        type = "int"             value =  "170"/>   	      <!-- 90, 170 -->
<param name= "cmd_vel_vision_output_topic"        type = "string"          value = "/cmd_vel/vision"/>
<param name= "vision_xte_topic"                   type = "string"          value = "/xte/vision"/>
<param name= "vision_xte_offset_topic"            type = "string"          value = "/xte/vision_offset"/> 

</node>
</launch>
